RecoParticleFlow/PFProducer/interface/PFEGammaHeavyObjectCache.h

```diff
@@ -2,23 +2,19 @@
 #define __RecoParticleFlow_PFProducer_pfEGHelpersHeavyObjectCache_h__
 
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
-#include "CondFormats/EgammaObjects/interface/GBRForest.h"
+#include "CommonTools/MVAUtils/interface/GBRForestTools.h"
 #include <memory>
 
 namespace pfEGHelpers {
   class HeavyObjectCache {
   public:
-    HeavyObjectCache(const edm::ParameterSet&);
```

Comment:
it looks like this change is beyond the minimal change requested for this PR.
The goal was to have only the lowPtGsfEle to be updated